doc/user/content/releases/v0.38.md

```diff
@@ -10,7 +10,7 @@ released: true
   system catalog views. This column provides an approximation of CPU utilization
   as a % of the total of compute workers.
 
-* Add [`mz_internal.mz_cluster_replica_frontiers`](/sql/system-catalog/mz_internal/#mz_cluster_replica_frontiers)
+* Add [`mz_internal.mz_cluster_replica_frontiers`](/sql/system-catalog/mz_internal/#mz_frontiers)
```

Comment:
I'm not sure what to do with links in release notes when the things they link to go away. This tries to keep the links somewhat useful, but should we just remove them instead?

Reply:
Removing is fine, yeah.